packages/pigeon/test/cpp_generator_test.dart

```diff
@@ -1018,6 +1018,250 @@ void main() {
     }
   });
 
+  test('flutter nullable arguments map correctly', () {
```

Comment:
Once a test gets over 100 lines of code, maybe we should have a docstring that explains the test a bit more.  I'm not sure what "map correctly" is immediately without reading through the test.